std/linked_list.zig

```diff
@@ -193,7 +193,11 @@ fn BaseLinkedList(comptime T: type, comptime ParentType: type, comptime field_na
         ///     A pointer to the new node.
         pub fn allocateNode(list: *Self, allocator: *Allocator) !*Node {
             comptime assert(!isIntrusive());
-            return allocator.create(Node);
+            return allocator.create(Node{
```

Comment:
It probably doesn't matter too much, but the right replacement is probably `return allocator.create(Node(undefined));`

Reply:
Cool, I was wondering about this!